certbot-apache/certbot_apache/tests/tls_sni_01_test.py

```diff
@@ -105,7 +105,7 @@ def test_mod_config(self):
         for achall in self.achalls:
             self.sni.add_chall(achall)
             z_domain = achall.response(self.auth_key).z_domain
-            z_domains.append(set([z_domain]))
+            z_domains.append(set([z_domain.decode('ascii')]))
```

Comment:
Yes this is safe. `z_domain` is `<32 char hex string>.<32 char hex string>.acme.invalid` ([source](https://github.com/certbot/certbot/blob/master/acme/acme/challenges.py#L399)).